packages/node_modules/@ciscospark/internal-plugin-board/src/board.js

```diff
@@ -128,6 +128,37 @@ const Board = SparkPlugin.extend({
   },
 
   /**
+   * Deletes a Channel from a Conversation
+   * @memberof Board.BoardService
+   * @param  {Conversation~ConversationObject} conversation
+   * @param  {Board~Channel} channel
+   * @returns {Promise}
+   */
+  deleteChannel(conversation, channel) {
+    return this.spark.request({
+      method: `PUT`,
+      uri: `${channel.aclUrl}/links`,
+      body: this._prepareDeleteChannel(conversation, channel)
+    })
+      .then((res) => res.body);
+  },
+
+  _prepareDeleteChannel(conversation, channel) {
+    // remove the ACL link between conversation and board
+    // remove conversation auth from board KRO in kms message
+    return {
+      aclLinkType: `INCOMING`,
+      linkedAcl: conversation.aclUrl,
+      kmsMessage: {
+        method: `delete`,
+        uri: `${channel.kmsResourceUrl}/authorizations?${querystring.stringify({authId: conversation.kmsResourceObjectUrl})}`
```

Comment:
note that request accepts a `qs` property and takes care of stringification if you give it a hash automatically

Reply:
this uri is for the kmsMessage though, would request take care of that?

Reply:
yep, keep 
```js
uri: `${channel.kmsResourceUrl}/authorizations`
```
then just put the querystring params in the `qs` hash.